content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,329 @@
+
+.. math::
+    \nabla ^2  \mathbf{E_x} + k^2 \mathbf{E_x} = 0
+    :label: E_wave_propagation_equation
+
+.. math::
+    \nabla ^2 \mathbf{H_y} + k^2 \mathbf{H_y} = 0
+    :label: H_wave_propagation_equation
+
+with k the wavenumber:
+
+.. math::
+    k = \sqrt{\omega ^2 \mu \varepsilon - i \omega \mu \sigma }
+    :label: kwavenumber
+
+
+In the ground, we usually assume that the displacement current is negligible, which means :math:`\sigma >> \omega \varepsilon`. In this case 
+
+.. math::
+    k_{ground} \simeq (1-i) \sqrt{ \frac{\omega \mu \sigma}{2} }
+    :label: kwavenumber_steadystate
+
+In the air, the conductivity is almost 0
+
+.. math::
+    k_{air} \simeq \omega \sqrt{ \mu_0 \varepsilon_0}
+    :label: kwavenumber_air
+
+Taking the problem from the point of view of the electric field, we know the equation :eq:`E_wave_propagation_equation` has a solution in the form of:
+
+.. math::
+    E_x (z) = U e^{i k z} + D e^{-i k z}
+    :label: Electric field components
+    
+.. math::
+    H_y (z) = \frac{(\nabla \times \mathbf{E_x})_y}{- i \omega \mu} = \frac{k}{ \omega \mu} (D e^{-i k z} - U e^{i k z}) = \frac{1}{Z} (D e^{-i k z} - U e^{i k z})
+    :label: Magnetic field components
+
+with :math:`\mathbf{E_x} = E_x \mathbf{\hat{x}}`  and U and D are the complex amplitudes of the Up and Down components of the field and Z the intrinsic impedance of the space.
+
+Writing the solution in the j-th layer (See :numref:`MTlayeredEarth`), we got:
+
+ .. math::
+    E_{x,j} (z) = U_j e^{i k (z-z_{j-1})} + D_j e^{-i k (z-z_{j-1})}
+    :label: Electric field components in layers
+    
+ .. math::
+    H_{y,j} (z) = \frac{1}{Z_j} (D_j e^{-i k (z-z_{j-1})} - U_j e^{i k (z-z_{j-1})})
+    :label: Magnetic field components in layers
+
+
+Which can be rewrite as:
+
+ .. math::
+    \left(\begin{matrix} E_{x,j} \\ H_{y,j} \end{matrix} \right) = \left(\begin{matrix} 1 & 1 \\ -\frac{1}{Z_j} & \frac{1}{Z_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = P_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right)
+    :label: Propagation matrix 
+
+The transition of the Up and Down component inside a layer can then be write as such
+
+ .. figure:: images/InsideLayer.png
+    :align: center
+    :scale: 100% 
+    :name: InsideLayer
+
+    Transition inside a layer, variables definition.
+
+
+.. math::
+    \left(\begin{matrix} U_j' \\ D_j' \end{matrix} \right)  = \left(\begin{matrix} e^{i k h_j} & 0 \\ 0 & e^{-i k h_j} \end{matrix} \right) \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+    = T_j \left(\begin{matrix} U_j \\ D_j \end{matrix} \right) 
+
+With the variables U, D, U' and D' define as in (:numref:`InsideLayer`)
+
+Using the continuity of the tangential \\(\\mathbf{E_x}\\) and \\(\\mathbf{H_y}\\) field at the interfaces, we find an iterative relation between the fields in consecutive layers:
```

Comment:
"tangential E_x , H_y" is a bit redundant, you could remove the _x, _y (ie . "tangential E, H") or remove the "tangential" (my preference - the first one). Also: could you use ``:math:`\mathbf{E_x}` `` instead (it renders the math in the pdf, the brackets won't)